x-pack/plugins/cases/server/client/utils.test.ts

```diff
@@ -5,113 +5,12 @@
  * 2.0.
  */
 
-import { SavedObjectsFindResponse } from 'kibana/server';
-import {
-  CaseConnector,
-  CaseType,
-  ConnectorTypes,
-  ESCaseConnector,
-  ESCasesConfigureAttributes,
-} from '../../common/api';
-import { mockCaseConfigure } from '../routes/api/__fixtures__';
+import { CaseConnector, CaseType, ConnectorTypes } from '../../common/api';
 import { newCase } from '../routes/api/__mocks__/request_responses';
-import {
-  transformCaseConnectorToEsConnector,
-  transformESConnectorToCaseConnector,
-  transformNewCase,
-} from '../common';
-import { getConnectorFromConfiguration, sortToSnake } from './utils';
+import { transformNewCase } from '../common';
+import { sortToSnake } from './utils';
 
 describe('utils', () => {
-  const caseConnector: CaseConnector = {
```

Comment:
I moved most of the functionality from the utils into the service layer's transform files and wrote the unit tests there.